CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## [Unreleased]
+- update tk-status to 1.1.1 which includes additional java memory info
 
 ## [1.7.31]
```

Comment:
I guess the switch from master to 1.7.x did something weird here (most recent version should be 1.7.38), but I guess it's fine.

Reply:
Whoops, I'll fiddle with it and see if I can get it right.